app/api/controllers/validators/ValidatorOps.scala

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api.controllers.validators
+
+import api.models.errors.MtdError
+import cats.data.Validated
+import cats.data.Validated.Valid
+import cats.implicits.toTraverseOps
+
+trait ValidatorOps {
```

Comment:
really, Validator and RulesValidator should extend this, so all the ValFactories get it "for free". I think its use will be clearer then too, as people might otherwise wonder why not just put it all in Validator.